pkg/syncthing/client.go

```diff
@@ -0,0 +1,63 @@
+package syncthing
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"net/http"
+	"path"
+	"time"
+)
+
+func NewApiClient() *http.Client {
+	return &http.Client{
+		Timeout: 60 * time.Second,
+	}
+}
+
+const (
+	GET  = "GET"
+	POST = "POST"
+	PUT  = "PUT"
+)
+
+// Makes API calls to the syncthing instance
+func (s *Syncthing) ApiCall(method string, url string, params map[string]string, body []byte) ([]byte, error) {
```

Comment:
Is this code borrowed from other repo? Please keep license head if so.

Reply:
Also the client style is not consistent with our current HTTP client

Reply:
Ok, yeah this is borrowed for now, I was going to ask you about the http client after you wake up.